platforms/yttrex/backend/bin/parser.ts

```diff
@@ -76,6 +77,10 @@ const run = async (): Promise<void> => {
       write: mongoW,
     };
 
+    const errorReporter = FixtureReporter(
+      path.resolve(process.cwd(), '__tests__/fixtures/htmls')
```

Comment:
Can we make the `"parsers/__tests__/fixtures"` string a constant variable or something?